marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -207,6 +221,101 @@ private void updateModelEntityTypes() {
         assertEquals("string", loadModel(getHubClient().getFinalClient()).get("definitions").get(MODEL_NAME).get("properties").get(ENTITY_PROPERTY_1).get("datatype").asText());
     }
 
+    private void addFacetableDateTimeProperty() {
+        Assumptions.assumeTrue(isVersionCompatibleWith520Roles());
+
+        String entityTypes = "[\n" +
+            "  {\n" +
+            "    \"entityName\": \"" + MODEL_NAME + "\",\n" +
+            "    \"modelDefinition\": {\n" +
+            "      \"Customer\": {\n" +
+            "        \"required\": [],\n" +
+            "        \"pii\": [\n" +
```

Comment:
I'm going to be picky since this is one of your first JUnit tests. :)

We want every test to be minimized to only the details that matter. In this case, we don't care about pii or required, so both should be removed (neither need to be there). And since elementRangeIndex is being set, facetable doesn't need to be set either. Also, collation is meaningless unless it's a string property, so that should be removed as well.